### PR TITLE
Changelog 23.9.0

### PR DESCRIPTION
**Pull request by jezdez**
## Description

✂️ snip snip ✂️ the making of a new release.

Xref #13080

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,113 @@
 [//]: # (current developments)
 
+## 23.9.0 (2023-09-27)
+
+### Enhancements
```
Comment (jezdez) [inline CHANGELOG.md:5]: I'll add a blurb about the solver switch here later today

File: .authors.yml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 - name: Michael Sarahan
   email: msarahan@continuum.io
-  num_commits: 1330
+  num_commits: 665
```
Comment (jezdez) [inline .authors.yml:3]: This is going down by using the current dev version of rever which implements counting correctly: https://github.com/regro/rever/pull/266

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,143 @@
...
+  - the `CONDA_SOLVER=classic` environment variable or
+  - running `conda config --set solver classic`.
+- All development of `conda-libmamba-solver` plugin happens in the [conda-libmamba-solver repo](https://github.com/conda/conda-libmamba-solver), including issue tracking.
+- The documentation of the `conda-libmamba-solver` plugin can be found on [conda.github.io/conda-libmamba-solver](https://conda.github.io/conda-libmamba-solver/).
+
+For more information about the `conda-libmamba-solver` rollout plan, please also see our [blog post from earlier this year](https://conda.org/blog/2023-07-05-conda-libmamba-solver-rollout).
+
+#### Context
+
+To address growth challenges within the conda ecosystem, the conda maintainers, alongside partners Anaconda, Quansight and QuantStack, introduced [a new conda dependency solver based on the Mamba project](https://mamba.readthedocs.io) in December 2022. A "solver" is a core component of most package managers, which calculates which dependencies (and which version) to also install, when a user requests to install a package from a package repository.
```
Comment (beeankha) [inline CHANGELOG.md:31]:
```suggestion
A "solver" is the core component of most package managers; it calculates which dependencies (and which version of those dependencies) to install when a user requests to install a package from a package repository. To address growth-related challenges within the conda ecosystem, the conda maintainers, alongside partners Anaconda, Quansight and QuantStack, introduced [a new conda dependency solver based on the Mamba project](https://mamba.readthedocs.io) in December 2022.
```

Changing some wording here and also the order of the sentences so it goes from high-level to more detailed context.

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,143 @@
...
+
+Since July 2023, that [`conda-libmamba-solver`](https://github.com/conda/conda-libmamba-solver) plugin has been included in and automatically installed with all major conda ecosystem installers (miniforge, miniconda, mambaforge and Anaconda Distribution), _with the default solver configuration unchanged_.
+
+### Enhancements
+
+* Improve speed of fish shell initialization. (#12811)
```
Comment (beeankha) [inline CHANGELOG.md:37]:
```suggestion
* Improve speed of `fish` shell initialization. (#12811)
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,143 @@
...
+* Directly suppress use of binstar (conda) token when fetching trust metadata. (#12889)
+* Adds a new "auth handler" plugin hook for conda. (#12911)
```
Comment (beeankha) [inline CHANGELOG.md:39]:
```suggestion
* Add a new "auth handler" plugin hook for conda. (#12911)
```

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,5 +1,143 @@
...
+* Lock index cache metadata by default. Added `--no-lock` option in case of
+  problems, should not be necessary. Older `--experimental=lock` no longer has
+  an effect. (#12920)
+* Add `context.register_envs` option to control whether to register environments
+  in `~/.conda/environments.txt` when they are created. Defaults to true. (#12924)
+* Inject a new detailed output verbosity level (i.e., the old debug level `-vv` is now `-vvv`). (#12985, #12977, #12420, #13036)
+* Add support for `truststore` to the `ssl_verify` config option, enabling conda to use the operating system certificate store (requires Python 3.10 or later). (#13075 and #13149)
+* Add `emscripten-wasm32` and `wasi-wasm32` platforms to known platforms. (#13095)
+* Adds the `py.typed` marker file to the `conda` package for compliance with PEP-561. (#13107)
+* Import boto3 only when s3 channels are used, saving startup time. (#12914)
```
Comment (beeankha) [inline CHANGELOG.md:49]:
```suggestion
* Import bBoto3 only when S3 channels are used, saving startup time. (#12914)
```

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,5 +1,143 @@
...
+
+### Bug fixes
+
+* When using pip dependencies with `conda env create`, check the directory permissions before writing to disk. (#11610)
+* Hide InsecureRequestWarning for jlap when CONDA_SSL_VERIFY=false, matching
+  non-jlap behavior. (#12731)
```
Comment (beeankha) [inline CHANGELOG.md:55]:
```suggestion
* Hide `InsecureRequestWarning` for JLAP when `CONDA_SSL_VERIFY=false`, matching
  non-JLAP behavior. (#12731)
```

File: CHANGELOG.md [hunk 8]
```diff
@@ -1,5 +1,143 @@
...
+* Disallow ability to create a conda environment with a colon in the prefix. (#13044)
+* Fix AttributeError logging response with nonexistent request, when using jlap
+  with file:/// URIs. (#12966)
```
Comment (beeankha) [inline CHANGELOG.md:58]:
```suggestion
* Fix `AttributeError` logging response with nonexistent request when using JLAP
  with `file:///` URIs. (#12966)
```

File: CHANGELOG.md [hunk 9]
```diff
@@ -1,5 +1,143 @@
...
+* Do not show progress bars in non interactive runs for cleaner logs. (#12982)
```
Comment (beeankha) [inline CHANGELOG.md:59]:
```suggestion
* Do not show progress bars in non-interactive runs for cleaner logs. (#12982)
```

File: CHANGELOG.md [hunk 10]
```diff
@@ -1,5 +1,143 @@
...
+* Fix S3 bucket name. (#12989)
+* Default `--json` and `--debug` to `NULL` so as to not overriding `CONDA_JSON` and `CONDA_DEBUG` environment variables. (#12987)
```
Comment (beeankha) [inline CHANGELOG.md:61]:
```suggestion
* Default `--json` and `--debug` to `NULL` so as to not override `CONDA_JSON` and `CONDA_DEBUG` environment variables. (#12987)
```

File: CHANGELOG.md [hunk 11]
```diff
@@ -1,5 +1,143 @@
...
+* ``XonshActivator`` now uses ``source-bash`` in non-interactive mode to avoid
+  side-effects from interactively loaded RC files
```
Comment (beeankha) [inline CHANGELOG.md:63]:
```suggestion
  side-effects from interactively loaded RC files. (#13012)
```

File: CHANGELOG.md [hunk 12]
```diff
@@ -1,5 +1,143 @@
...
+* Fix conda remove --all --json output. (#13019)
```
Comment (beeankha) [inline CHANGELOG.md:64]:
```suggestion
* Fix `conda remove --all --json` output. (#13019)
```

File: CHANGELOG.md [hunk 13]
```diff
@@ -1,5 +1,143 @@
...
+* Update test data to stop triggering security scanners false-positives. (#13034)
```
Comment (beeankha) [inline CHANGELOG.md:65]:
```suggestion
* Update test data to stop triggering security scanners' false-positives. (#13034)
```

File: CHANGELOG.md [hunk 14]
```diff
@@ -1,5 +1,143 @@
...
+* Fix performance regression of basic commands (e.g., `conda info`) on WSL. (#13035)
+* Configure conda to ignore "Retry-After" header to avoid the scenarios when this value is very large and causes conda to hang indefinitely. (#13050)
+* Treat `JSONDecodeError` on `repodata.info.json` as a warning, equivalent to a
+  missing `repodata.info.json`. (#13056)
+* Fix sorting error for `conda config --show-sources --json`. (#13076)
+* Catch OSError in `find_commands` to account for incorrect PATH entries on
```
Comment (beeankha) [inline CHANGELOG.md:71]:
```suggestion
* Catch `OSError` in `find_commands` to account for incorrect `PATH` entries on
```

File: CHANGELOG.md [hunk 15]
```diff
@@ -1,5 +1,143 @@
...
+* Removed `conda.core.index.check_whitelist()`. (#12948)
+* Removed `conda.core.solve._get_solver_class()`. (#12948)
+* Removed `conda.core.subdir_data.read_mod_and_etag()`. (#12948)
+* Removed `conda.gateways.repodata.RepodataState.load()`. (#12948)
+* Removed `conda.gateways.repodata.RepodataState.save()`. (#12948)
+* Removed `conda.lock` module. (#12948)
+* Removed `conda_env.cli.common.stdout_json()`. (#12948)
+* Removed `conda_env.cli.common.get_prefix()`. (#12948)
+* Removed `conda_env.cli.common.find_prefix_name()`. (#12948)
+* Remove import of deprecated cgi module by deprecating ftp STOR support.
```
Comment (beeankha) [inline CHANGELOG.md:100]:
```suggestion
* Remove import of deprecated `cgi` module by deprecating `ftp STOR` support.
```

File: CHANGELOG.md [hunk 16]
```diff
@@ -1,5 +1,143 @@
...
+  (#13013)
+* Require `boto3` for s3 support and drop support for the older `boto` as it
```
Comment (beeankha) [inline CHANGELOG.md:102]:
```suggestion
* Require `boto3` for S3 support and drop support for the older `boto` as it
```

File: CHANGELOG.md [hunk 17]
```diff
@@ -1,5 +1,143 @@
...
+  doesn't support our minimum required version of Python. (#13112)
+* Reduce startup delay from deprecations module by using `sys._getframe()`
+  instead of `inspect.stack()`. (#12919)
+
+### Other
+
+* Use Ruff linter in pre-commit configuration (#12279)
+* Remove unused cache_path arguments from RepoInterface/JlapRepoInterface;
```
Comment (beeankha) [inline CHANGELOG.md:110]:
```suggestion
* Remove unused `cache_path` arguments from `RepoInterface`/`JlapRepoInterface`;
```

**Review by beeankha — APPROVED** [review]

**Review by kenodegard — APPROVED** [review]

**Pull request merged**